util/tls/tls_socket.h

```diff
@@ -46,6 +46,14 @@ class TlsSocket : public FiberSocketBase {
     return next_sock_->IsOpen();
   }
 
+  void set_timeout(uint32_t msec) final override {
```

Comment:
have you checked it's working?

Reply:
No, I don't know how to check. If you know how to do it, I will check

Reply:
there is `examples/https_client_cli.cc` that you can run . add a timeout setting and see if it works.
that's a minimum requirement. You can also introduce a unit test with tls client/server  but that's a bigger chunk of work.

Reply:
I've checked that it works